src/components/governance/PageProposal.vue

```diff
@@ -2,9 +2,12 @@
   <TmPage data-title="Proposal" hide-header class="small">
     <TmDataLoading
       v-if="
-        $apollo.queries.proposal.loading || $apollo.queries.parameters.loading
+        $apollo.queries.proposals.loading ||
+          $apollo.queries.proposal.loading ||
+          $apollo.queries.parameters.loading
       "
     />
+    <TmDataNotFound v-else-if="!found" />
```

Comment:
if it's not found, should it show the 404 page instead?

Reply:
mmm I dont think it's better to show the 404 page, because the page exists, the resource you are looking for doesn't exists.

Reply:
yep, I have the same feeling. I will merge and we can discuss this later again.